data/json/recipes/tools/tools_hand.json

```diff
@@ -1589,10 +1624,13 @@
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
     "difficulty": 3,
-    "time": "15 m",
+    "time": "30 m",
     "autolearn": true,
+    "qualities": [ [ { "id": "SIEVE", "level": 1 } ], [ { "id": "HAMMER", "level": 2 } ] ],
```

Comment:
This item 
```
  {
    "id": "casting_mold",
    "type": "TOOL",
    "name": { "str": "sand casting mold" },
    "description": "A mold for casting molten metals into different shapes.",
    "weight": "540 g",
    "volume": "400 ml",
    "longest_side": "100 cm",
    "price": 800,
    "price_postapoc": 10,
    "to_hit": -2,
    "material": [ "clay", "wood" ],
    "symbol": ";",
    "color": "brown",
    "melee_damage": { "bash": 10 }
  },
```
Make zero sense.  It's one meter long and from all the examples I can find most casting molds are twice as long as they are high and wide.  So the volume of this object would be 250 Liters which means it will need a minimum of 240 liters of sand.  which means the weight of the item will also go way up.

Reply:
I agree the existing item is a bit silly, I'll have a go at fixing it too.

Reply:
My proposal would be to introduce 3 sizes of sand casting mold. I've looked through the recipes that need one today, taking into consideration several also require long bits of wood (not part of the mould):

- Ensure there is 2cm of clearance each side, each dimension, for the mould
- Estimate the volume and length for the metal parts, e.g. just the head of the pick axe
- Larger moulds would also work for smaller items

This gives three sizes roughly capable of housing a mould thus:
- **small**: 5L, ~30cm length and ~13cm width/height. This would work for nail, nail puller, glass shard, shears, maybe some other 'heads' (i need to check each item more)
- **normal/medium**: ~12L, ~55cm length and ~14cm width/height. Covers tongues, bronze sword, weather glass, maybe any larger 'heads'
- **large**: ~42L, ~65cm length and ~25cm width/height: Covers tiny engine block, pick axe head.

The actual dimensions would be a bit bigger to cover some kind of wooden housing frame (using the 2x4 and nails).
Does that sound reasonable?

Reply:
i've had a go anyway :-)